src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -140,7 +140,20 @@ public virtual void Init(IProviderConfiguration providerCfg, string providerName
             {
                 var bufferPool = new FixedSizeObjectPool<FixedSizeBuffer>(adapterSettings.CacheSizeMb, () => new FixedSizeBuffer(1 << 20));
                 var timePurge = new TimePurgePredicate(adapterSettings.DataMinTimeInCache, adapterSettings.DataMaxAgeInCache);
-                CacheFactory = (partition,checkpointer,cacheLogger) => new EventHubQueueCache(checkpointer, bufferPool, timePurge, cacheLogger, this.SerializationManager);
+                if (adapterSettings.SlowConsumingMonitorThreshold > 0)
```

Comment:
We tend to use nullable values for optional settings.  Suggest SlowConsumingMonitorThreshold be nullable rather than using it's value to detect if it's set or not.